src/clims/services/extensible_service_api.py

```diff
@@ -103,3 +104,25 @@ def _get_filter_arguments(self, **kwargs):
             else:
                 get_args['archetype__{}'.format(key)] = value
         return get_args
+
+    def _get_values_qs(self, property, extensible_type):
+        extensible_type_name = extensible_type.type_full_name_cls()
+        property_query_set = ExtensibleProperty.objects.filter(
+            extensible_property_type__extensible_type__name=extensible_type_name,
+            extensible_property_type__name=property).select_related('extensible_property_type')
+        return property_query_set
+
+    def get_values_of_property(self, property, extensible_type):
+        property_query_set = self._get_values_qs(property, extensible_type)
+        extensible_type = property_query_set[0].extensible_property_type
```

Comment:
Perhaps this is wrong spelling? extenstible_type already means something and it's not extensible_property_type.